warehouse/seeds/gtfs_schedule_validation_code_descriptions.csv

```diff
@@ -0,0 +1,53 @@
+Type,Name,Human_Readable_Description,"Critical_for_reporting",What_to_Report
```

Comment:
same Q: is it possible to comment where this came from, especially version information?